### PR TITLE
Add cobol pl1 lang servers

### PR DESCRIPTION
**Pull request by PrasangAPrajapati**
Add IBM Enterprise COBOL and PL/1 Language Servers

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1368,7 +1409,7 @@ <h4>LSP clients</h4>
 			</tr>
 			<tr>
 			    <th>IntelliJ / JetBrains IDEs</th>
-			    <td><a href="https://github.com/gtache">Guillaume Tâche</a></td>
+			    <td><a href="https://github.com/gtache">Guillaume T�che</a></td>
```
Comment (keegancsmith) [inline index.html]: I'm not sure what your editor did, but it seems all the line with non-ascii characters in it have changed? Can you undo these changes, or explain why it is fine?

Comment (PrasangAPrajapati) [reply index.html]: @keegancsmith I don't know what happened but which encoding have you guys used ?

Comment (keegancsmith) [reply index.html]: utf-8
